data/ignition/systemd/units/assisted-service-db.service

```
@@ -0,0 +1,21 @@
+[Unit]
+Description=Assisted Service database
+Wants=network.target
+After=network-online.target
+RequiresMountsFor=%t/containers
+BindsTo=assisted-service-pod.service
+After=assisted-service-pod.service
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+Restart=on-failure
+TimeoutStopSec=300
+ExecStartPre=/bin/rm -f %t/%n.ctr-id
+ExecStart=/usr/bin/podman run --cidfile=%t/%n.ctr-id --cgroups=no-conmon --log-driver=journald --rm --pod-id-file=%t/assisted-service-pod.pod-id --sdnotify=conmon --replace -d --name=db --env-file=/usr/local/share/assisted-service/assisted-db.env quay.io/centos7/postgresql-12-centos7:latest
```

Maybe we can put something a bit more discoverable as a name.

```suggestion
ExecStart=/usr/bin/podman run --cidfile=%t/%n.ctr-id --cgroups=no-conmon --log-driver=journald --rm --pod-id-file=%t/assisted-service-pod.pod-id --sdnotify=conmon --replace -d --name=assisted-pgsql --env-file=/usr/local/share/assisted-service/assisted-db.env quay.io/centos7/postgresql-12-centos7:latest
```

---

Added another commit to change the names (previous names were the same ones used in the kube-play pod).